packages/crypto/src/keypair.ts

```diff
@@ -2,57 +2,112 @@ import nacl from 'tweetnacl'
 import multibase from 'multibase'
 import { Private, Public } from './identity'
 import { encodePublicKey, encodePrivateKey, KeyType } from './proto.keys'
+// eslint-disable-next-line import/no-cycle
 import { publicKeyToString, privateKeyFromString, publicKeyBytesFromString, encrypt, decrypt } from './utils'
 
+/**
+ * Encode the given PrivateKey to its base-32 encoded multibase representation.
+ * @param key The input PrivateKey.
+ */
 export function privateKeyToString(key: PrivateKey) {
   const encoded = multibase.encode('base32', key.bytes as Buffer)
   return new TextDecoder().decode(encoded)
 }
 
+/**
+ * Default implementation of the {@link Public} interface, with encryption
+ * support.
+ * In theory, RSA, ed25519, and secp256k1 key types (and more) should be
+ * supported, although currently only ed25519 has full verify and encrypt
+ * capabilities.
+ * @todo Support additional key types by default. For now, sticking to ed25519
+ * keeps our bundle smaller, but if needed, additional types can be added via
+ * libp2p-crypto or tweetnacl.
+ * @todo Separate out the generic PublicKey interface to include signing and
```

Comment:
let's try to stick to the api-extractor supported tags, https://api-extractor.com/pages/tsdoc/doc_comment_syntax/ to ensure that rendering works as expected